unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/MainChatNotificationsComponentView.cs

```diff
@@ -0,0 +1,221 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using DCL.Interface;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MainChatNotificationsComponentView : BaseComponentView
+{
+    [SerializeField] RectTransform chatEntriesContainer;
+    [SerializeField] GameObject chatNotification;
+    [SerializeField] ScrollRect scrollRectangle;
+    [SerializeField] Button notificationButton;
+    [SerializeField] public TMP_Text notificationMessage;
+
+    public ChatNotificationController controller;
+
+    private const string NOTIFICATION_POOL_NAME_PREFIX = "NotificationEntriesPool_";
+    private const int MAX_NOTIFICATION_ENTRIES = 30;
+
+    public event Action<string> OnClickedNotification;
+    public bool areOtherPanelsOpen = false;
+
+    internal Queue<PoolableObject> poolableQueue = new Queue<PoolableObject>();
+    internal Queue<ChatNotificationMessageComponentView> notificationQueue = new Queue<ChatNotificationMessageComponentView>();
+
+    private Pool entryPool;
+    private bool isOverMessage = false;
+    private bool isOverPanel = false;
+    private int notificationCount = 1;
+
+    public static MainChatNotificationsComponentView Create()
+    {
+        return Instantiate(Resources.Load<MainChatNotificationsComponentView>("SocialBarV1/ChatNotificationHUD"));
+    }
+
+    public void Initialize(ChatNotificationController chatController)
```

Comment:
this could be replaced for `Awake` method following the principle that the views should have their basic functionality working on their own